src/activity_item_handling.cpp

```diff
@@ -1421,10 +1374,11 @@ static void add_basecamp_storage_to_loot_zone_list( zone_manager &mgr, const tri
     if( npc *const guy = dynamic_cast<npc *>( &you ) ) {
         map &here = get_map();
         if( guy->assigned_camp &&
-            mgr.has_near( zone_type_CAMP_STORAGE, here.getglobal( src_loc ), ACTIVITY_SEARCH_DISTANCE ) ) {
+            mgr.has_near( zone_type_CAMP_STORAGE, here.getglobal( src_loc ), ACTIVITY_SEARCH_DISTANCE ),
+            _fac_id( you ) ) {
```

Comment:
It looks like parenthesis is positioned incorrectly in here. Should be:

![image](https://user-images.githubusercontent.com/16213433/185810803-b43cda50-f7c5-4f28-a590-2014a0b6a047.png)